CMakeLists.txt

```diff
@@ -122,6 +122,10 @@ set(IGN_RENDERING_VER ${ignition-rendering4_VERSION_MAJOR})
 ign_find_package(ignition-math6 REQUIRED COMPONENTS eigen3 VERSION 6.6)
 set(IGN_MATH_VER ${ignition-math6_VERSION_MAJOR})
 
+#--------------------------------------
+# Find ignition-tools
+ign_find_package(ignition-tools QUIET)
```

Comment:
Can we make this required? This should work:

```
ign_find_package(ignition-tools
  REQUIRED
  PKGCONFIG "ignition-tools")
```

See https://github.com/ignitionrobotics/ign-tools/issues/23

Reply:
added in 0dfa9aa175adf053c068ac087e8311b09e9897ad